library/general/containerlimits/template.yaml

```diff
@@ -3,8 +3,11 @@ kind: ConstraintTemplate
 metadata:
   name: k8scontainerlimits
   annotations:
-    description: Requires containers to have memory and CPU limits set and within
-      a specified maximum amount.
+    description: >-
+      Requires containers to have memory and CPU limits set and constrains
+      limits to set maximum values.
```

Comment:
double use of the word "set" with different meanings can be confusing.

"to be within the specified maximum values"

Reply:
Done